### PR TITLE
Test new action

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Once this looks good to you two we can run it and see what happens. This won't be merged as it's just a test

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -40,22 +40,22 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
-      - uses: NYCPlanning/action-library-archive@v1.1
+      - uses: NYCPlanning/action-library-archive@v1.2-alpha
         if: matrix.dataset != 'pluto_corrections'
         with:
           name: ${{ matrix.dataset }}
           s3: true
-          latest: true
+          latest: false
```
Comment (td928) [inline .github/workflows/publish.yml:48]: I guess I haven't thought about the DO side. I think it is definitely smart to not touch the latest for any of the datasets.

**Review by td928 — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (SashaWeinstein) [reply .github/workflows/publish.yml:48]: Yea that was my thinking. Should be easy to fix by running publishing from the main branch if we do mess something up though

**Comment by SashaWeinstein** [comment]
Ok so I wanted to report back on the behavior that I'm seeing that I can't really figure out.

### Run with v1.2
[The first publish action](https://github.com/NYCPlanning/db-pluto/actions/runs/3010444718)  was with the new version of the data library action as we had planned. All the jobs failed with this error 
`python: can't open file 'script.py': [Errno 2] No such file or directory`
which I don't understand. The container was built off the image correctly, and the tag worked when I pulled a container down on my local. 

### Run with v1.1
[The second run](https://github.com/NYCPlanning/db-pluto/actions/runs/3010499228) uses old version (v1.1) of the data library action. Github reports that this one worked, with all 5 datasets getting green checkmarks. I went to digital ocean and saw recently updated 22v2 data in data library for `dcp_mappluto` and `pluto_corrections`, but no 22v2 for `dcp_mappluto_wi`, `dcp_pluto`, or `pluto_mappluto_clipped`.

I think the good news is that v1.1 points to the latest image, so the new image I pushed works at least sometimes. The bad news is that the new image missed some datasets, and the v1.2 of the data library action doesn't work at all. 

Hopefully I will think of how to proceed when I look at this tomorrow, would appreciate any thoughts that folks had

**Comment by SashaWeinstein** [comment]
Oh wait an important detail here is that this action passes the name instead of the path. So this is actually what we would expect to see, I didn't realize that change was already in there. Tomorrow I will be able to make more progress

**Comment by SashaWeinstein** [comment]
Another important detail here is that when I pushed the new image it didn't overwrite `ubuntu-latest`. You can see on [docker hub](https://registry.hub.docker.com/r/nycplanning/library/tags) that ubuntu latest was pushed by Baiyue 9 months ago. So don't need to worry about existing publishing workflows pointing to new version

**Pull request closed without merging**